core/scss/ng/settings/settings.scss

```diff
@@ -8101,7 +8101,8 @@ $blockquote-padding: rem-calc(9 20 0 19) !default;
 $blockquote-border-width: 4px !default;
 $blockquote-padding: 0.5rem 1rem calc(0.25rem + #{$blockquote-border-width})
   1rem !default;
-$blockquote-border: $blockquote-border-width solid $md-gray-50 !default;
+$blockquote-border: $blockquote-border-width solid !default;
+$blockquote-border-color: $md-gray-50 !default;
```

Comment:
Assuming a neutral color works in this use case instead of a css var?

Reply:
Yes.